packages/dialect-react-ui/tailwind.config.js

```diff
@@ -7,5 +7,6 @@ module.exports = {
     extend: {},
   },
   plugins: [],
+  prefix: 'dt-',
   important: '.dialect',
```

Comment:
Since dt prefix is here now, we can safely remove the `important` rule here too.